src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -1,805 +1,822 @@
+        await: () => this.pageMode === 'patient' ? [] : [this.derivedPatientId],
+        invoke: async() => fetchClinicalDataForPatient(this.studyId, this.patientId),
+        default: []
+    });
+
+    readonly samples = remoteData(
+        async() => fetchSamplesForPatient(this.studyId, this._patientId, this.sampleId),
+        []
+    );
+
+    readonly samplesWithoutCancerTypeClinicalData = remoteData({
+        await: () => [
+            this.samples,
+            this.clinicalDataForSamples
+        ],
+        invoke: async () => findSamplesWithoutCancerTypeClinicalData(this.samples, this.clinicalDataForSamples)
+    }, []);
+
+    readonly studiesForSamplesWithoutCancerTypeClinicalData = remoteData({
+        await: () => [
+            this.samplesWithoutCancerTypeClinicalData
+        ],
+        invoke: async () => fetchStudiesForSamplesWithoutCancerTypeClinicalData(this.samplesWithoutCancerTypeClinicalData)
+    }, []);
+
+    readonly studies = remoteData({
+        invoke: async()=>([await client.getStudyUsingGET({studyId: this.studyId})])
+    }, []);
+
+    readonly studyIdToStudy = remoteData({
+        await: ()=>[this.studies],
+        invoke:()=>Promise.resolve(_.keyBy(this.studies.result, x=>x.studyId))
+    }, {});
+
+    readonly cnaSegments = remoteData({
+        await: () => [
+            this.samples
+        ],
+        invoke: () => fetchCopyNumberSegments(this.studyId, this.sampleIds)
+    }, []);
+
+    readonly pathologyReport = remoteData({
+        await: () => [this.derivedPatientId],
+        invoke: () => {
+            // only check path report for tcga studies
+            if (this.studyId.toLowerCase().indexOf('tcga') > -1) {
+                const pathLinkUrl = "https://raw.githubusercontent.com/inodb/datahub/a0d36d77b242e32cda3175127de73805b028f595/tcga/pathology_reports/symlink_by_patient";
+                const rawPdfUrl = "https://github.com/inodb/datahub/raw/a0d36d77b242e32cda3175127de73805b028f595/tcga/pathology_reports";
+                const reports: PathologyReportPDF[] = [];
+
+                // keep checking if patient has more reports recursively
+                function getPathologyReport(patientId:string, i:number):any {
+                    return request.get(`${pathLinkUrl}/${patientId}.${i}`).then(function(resp){
+                            // add report
+                            const pdfName: string = resp.text.split('/')[1];
+                            reports.push({name: `${pdfName}`, url: `${rawPdfUrl}/${pdfName}`});
+                            // check if patient has more reports
+                            return getPathologyReport(patientId, i+1);
+                        }, () => reports);
+                }
+
+                return getPathologyReport(this.patientId, 0);
+            } else {
+                return Promise.resolve([]);
+            }
+        },
+        onError: (err: Error) => {
+            // fail silently
+        }
+
```

Comment:
looks like you have reformatted this file, causing every line to be updated?

Reply:
the file had carriage return in windows format ^M, got red ! warning in WebStorm, so changed to linux CRLF. Should I roll back the change?

Reply:
were you able to roll this back?   I just checked PatientVIewPageStore in master and webstorm says it's set to LF, like all rest of files, so perhaps try rebasing?

Reply:
will do a rebase. thanks!